src/lib/arch/aarch64/ExceptionHandler.cc

```diff
@@ -586,6 +592,32 @@ bool ExceptionHandler::init() {
         stateChange = {ChangeType::REPLACEMENT, {R0}, {0ull}};
         break;
       }
+      case 278: {  // getrandom
+        // Write <buflen> random bytes to buf
+        uint64_t bufPtr = registerFileSet.get(R0).get<uint64_t>();
+        size_t buflen = registerFileSet.get(R1).get<size_t>();
+
+        char buf[buflen];
+        for (size_t i = 0; i < buflen; i++) {
+          buf[i] = (uint8_t)rand();
```

Comment:
To make this truly random, it should probably be seeded... 
From C++ Reference : `srand (time(NULL));` will initialise a random seed.

Reply:
Seeded with clock() because the resolution of time() is too large to give different random bytes on sequential syscalls